qbft/spectest/tests/messages/signature_too_short.go

```diff
@@ -9,13 +9,9 @@ import (
 
 // SignedMsgSigTooShort tests SignedMessage len(signature) < 96
 func SignedMsgSigTooShort() *tests.MsgSpecTest {
-	msg := testingutils.SignQBFTMsg(testingutils.Testing4SharesSet().Shares[1], types.OperatorID(1), &qbft.Message{
-		MsgType:    qbft.CommitMsgType,
-		Height:     qbft.FirstHeight,
-		Round:      qbft.FirstRound,
-		Identifier: []byte{1, 2, 3, 4},
-		Data:       testingutils.CommitDataBytes([]byte{1, 2, 3, 4}),
-	})
+	ks := testingutils.Testing4SharesSet()
```

Comment:
irrelevant with ssz

Reply:
@alonmuroch What do you think about changing the `Signature` type from `[]byte` to `[96]byte`? This won't allow `Signature` to have a wrong length

Reply:
If we leave it as `[]byte` we are still able to assign something non-96-bytes-long to the field, so I think we still can test whether we get an ssz-marshal/unmarshal error on incorrect length or not